dev-lang/ocaml/ocaml-4.09.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+HOMEPAGE="https://ocaml.org/"
+SRC_URI="https://github.com/ocaml/ocaml/archive/${PV}.tar.gz -> ${P}.tar.gz"
+DESCRIPTION="Programming language supporting functional, imperative & object-oriented styles"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="emacs flambda latex ncurses spacetime X xemacs"
+
+RDEPEND="sys-libs/binutils-libs:=
+	ncurses? ( sys-libs/ncurses:0= )
+	spacetime? ( sys-libs/libunwind:= )
+	X? ( x11-libs/libX11 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+PDEPEND="emacs? ( app-emacs/ocaml-mode )
+	xemacs? ( app-xemacs/ocaml )"
+
+src_configure() {
+	local opt=(
+		--bindir="${EPREFIX}/usr/bin"
+		--libdir="${EPREFIX}/usr/$(get_libdir)/ocaml"
+		--mandir="${EPREFIX}/usr/share/man"
+		--prefix="${EPREFIX}/usr"
+		$(use_enable flambda)
+		$(use_enable spacetime)
+	)
+	sed -i -e 's/^P ?=/P =/' stdlib/StdlibModules || die "P fix failed"
+	econf ${opt[@]}
+}
+
+src_compile() {
+	emake world.opt
+}
+
+src_test() {
+	# OCaml tests only work when run sequentially
+	make -j1 -C testsuite all
```

Comment:
why not `emake`?

Reply:
On my machine, `emake` would result in `make -j8 -j1 ...`. It would still only use sequential jobs, but did not look right to me. Should I use emake anyway?